internal/k8s/controller.go

```diff
@@ -199,6 +199,7 @@ type NewLoadBalancerControllerInput struct {
 	IsPrometheusEnabled          bool
 	IsLatencyMetricsEnabled      bool
 	IsTLSPassthroughEnabled      bool
+	TLSPassthroughPort           int
```

Comment:
@haywoodsh  is there a way to assert config when `IsTLSPassthroughEnabled=false` and `TLSPassthroughPort` is set?